### DISCUSSION TITLE
Is there a way to change the controller inputs?

### DISCUSSION THREAD

**Discussion by GanzSimpel33** [DaXcess/LCVR#55] [General]
Hey there I’ve been loving the mod a lot lately but sadly since I’m using a Vive Headset with Vive controllers I’ve noticed that I have to use the keyboard to drop items and jump since there are no A,B,Y,X buttons. So I was wondering if there’s a way to change the input settings to customize the controller layout for these controllers and possibly others?

**Comment by DaXcess** [comment]
The newest versions of the mod contain a configuration option called `ControllerBindingsOverrideProfile`. You can change this to any of the available profiles available on the [LCVR Controller Profiles](https://github.com/DaXcess/LCVR-Controller-Profiles) page, though there are no Vive binding overrides as of now, since these bindings are crowdsourced from people with these headsets/controllers and the know how to create bindings for them.
